test/functionalTest/cases/1164_v2_update_entities_attr_empty_value/update_entities_attr_empty_value.test

```diff
@@ -41,14 +43,27 @@ orionCurl --url /v2/entities --payload "$payload" --json
 echo
 echo
 
-echo "02. PUT /v2/entities/E1/attrs/A1 (empty)"
-echo "==============================================="
+echo "02. GET /v2/entities/E1/attrs/A1"
+echo "================================"
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo "03. PUT /v2/entities/E1/attrs/A1 (empty)"
+echo "========================================"
 payload='{ }'
 orionCurl --url /v2/entities/E1/attrs/A1 --payload "$payload" --json -X PUT
```

Comment:
Perhaps we should create an issue (about PUT on /v2/entities/E1/attrs/A1) ... 
This is not really part of the PR, just 'other' functests that needed correction ...